packages/mask/src/plugins/Wallet/SNSAdaptor/SelectProviderDialog/PluginProviderRender.tsx

```diff
@@ -174,6 +181,9 @@ export function PluginProviderRender({
                             .map((network, i) => (
                                 <NetworkItem
                                     key={i}
+                                    disabled={Boolean(
+                                        supportedNetworkList && !supportedNetworkList?.includes(network.type),
```

Comment:
```suggestion
                                       !supportedNetworkList?.includes(network.type),
```

Reply:
No, `supportedNetworkList && !supportedNetworkList?.includes(network.type)` is not identify to `!supportedNetworkList?.includes(network.type)`